047-TrafficControlWithDapr/Student/Challenge-00.md

```diff
@@ -0,0 +1,323 @@
+
+### Install local prerequisites
+  
+1.  Install all the prerequisites listed below and make sure they're working correctly:
+
+    - Git ([download](https://git-scm.com/))
+    - .NET 5 SDK ([download](https://dotnet.microsoft.com/download/dotnet/5.0))
+    - Visual Studio Code ([download](https://code.visualstudio.com/download)) with the following extensions installed:
+      - [C#](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)
+      - [REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client)
+    - Docker for desktop ([download](https://www.docker.com/products/docker-desktop))
+    - Dapr CLI and Dapr runtime ([instructions](https://docs.dapr.io/getting-started/install-dapr-selfhost/))
+    - Install Azure CLI
+      - Linux ([instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt))
+      - macOS ([instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-macos))
+      - Windows ([instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli))
+    - Install Bicep extension for VS Code ([instructions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep))   
+    - If you're running Windows, you'll need to install a **bash shell** to run some of the commands. Install either the [Git Bash](https://git-scm.com/downloads) client or the [Windows Subsystem for Linux 2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+
+    Make sure the following minimum software versions are installed by executing the commands in the following table:
+
+    | Software             | Version | Command Line       |
+    | -------------------- | ------- | ------------------ |
+    | Dapr runtime version | v1.2.2  | ```dapr --version```   |
+    | Dapr CLI version     | v1.2.0  | ```dapr --version```   |
+    | DotNet version       | 5.0.302 | ```dotnet --version``` |
+    | azure-cli            | 2.24.0  | ```az --version```     |
+
+### Create Azure Resources
+
+This WhatTheHack will create the following Azure resources in your Azure resource group. Make sure you can create the following:
+
+- Application Insights
+- Azure Kubernetes Service
+- Azure Container Registry
+- Event Hub Namespace
+- IoT Hub
+- KeyVault
+- Log Analytics Workspace
+- Logic App (with the Office 365 activity for sending email)
+- Azure Cache for Redis
+- Storage Account
+- Service Bus Namespace
+
+*If you can't instantiate some of these resources, you won't be able to complete the part of the challenge that uses them, but you may still be able to complete the other challenges*
+
+#### Special Considerations for Azure Kubernetes Service (AKS)
+
+- AKS requires the ability to create a public IP address. This is blocked by many organizations. You will either need to get an exception or have an admin create the AKS cluster for you.
```

Comment:
Let's change the 2nd sentence to: "This may be blocked by some organizations."
(Hopefully, this makes it sound less onerous.)